r-package/grf/vignettes/rate.Rmd

```diff
@@ -0,0 +1,216 @@
+---
+title: "Assesing heterogeneity with RATE"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{rate}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+set.seed(123)
+options(warn = -1)
+```
+
+```{r setup}
+library(grf)
+library(ggplot2)
+```
+
+This vignette gives a brief introduction to how the *Rank-Weighted Average Treatment Effect* (*RATE*) available in the function `rank_average_treatment_effect` can be used to evaluate how good estimates of conditional average treatment effects or other treatment *rules* are at distinguishing subpopulations with different treatment effects, or whether there is any notable heterogeneity present. The second part of the vignette gives a worked example from a medical setting using (synthetic) data from the SPRINT and ACCORD hypertension drug trials. For complete details we refer to the [RATE paper](https://arxiv.org/abs/2111.07966).
+
+## Treatment prioritization rules
+We are in the familiar experimental setting (or unconfounded observational study) and are interesting in the problem of determining which individuals to assign a binary treatment $W=\{0, 1\}$ and the associated value of adhering to this treatment assignment. Given some subject characteristics $X_i$ we have access to a subject-specific *prioritization rule* $S(X_i)$ which assigns scores to subjects. This prioritization rule should give a high score to units which we believe to have a large benefit of treatment and a low score to units with a low benefit of treatment. By benefit of treatment, we mean the difference in outcomes $Y$ from receiving the treatment given some subject characteristics $X_i$, as given by the conditional average treatment effect (CATE)
```

Comment:
"prioritization rule" --> "treatment prioritization rule" to be consistent with the header